src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/SurveyServiceImpl.java

```diff
@@ -3,29 +3,35 @@
 import java.util.UUID;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
 import uk.gov.ons.ctp.response.collection.exercise.domain.Survey;
 import uk.gov.ons.ctp.response.collection.exercise.repository.SurveyRepository;
 import uk.gov.ons.ctp.response.collection.exercise.service.SurveyService;
+import uk.gov.ons.response.survey.representation.SurveyDTO;
 
 /**
  * The implementation of the SampleService
  *
  */
 @Service
+@Qualifier("database")
 public class SurveyServiceImpl implements SurveyService {
```

Comment:
Presumably we can delete this class now?

Reply:
I'd like to keep it in for now - in case unforseen issues require a revert